core-plugins/src/main/java/co/cask/hydrator/plugin/common/CubeSinkConfig.java

```diff
@@ -46,6 +46,11 @@
     "http://docs.cask.co/cdap/current/en/developers-manual/building-blocks/datasets/cube.html#cube-configuration " +
     "for more information.")
   @Nullable
+  String aggregations;
```

Comment:
yeah, we can use an already existing cube. in that case we wouldn't need to provide aggregations.